Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1391,7 +1394,10 @@ static bool isContextMenuEvent(const WebMouseEvent& event)
 void UnifiedPDFPlugin::scrollToPointInPage(IntPoint pointInPDFPageSpace, PDFDocumentLayout::PageIndex pageIndex)
 {
     auto pointInContentsSpace = convertFromPageToContents(pointInPDFPageSpace, pageIndex);
-    scrollToPositionWithoutAnimation(pointInContentsSpace);
+    auto scaleFactor = this->scaleFactor() * documentFittingScale();
+    auto xMarginOffset = m_documentLayout.documentMargin.width()  * scaleFactor;
```

Comment:
Can you replace all of this with something like 

`pointsInContentsSpace - m_documentLayout.documentMargin.scaled(scaleFactor)`?

Reply:
^ +1